### ISSUE TITLE
Segmentation fault error and Symmetry error

### ISSUE THREAD

**Issue by pxlxingliang** [deepmodeling/abacus-develop#2474]
### Details

Here are a batch of examples that have appeared Segmentation fault error and Symmetry error.

The segmentation fault is like:
```
 UNIFORM GRID DIM     : 72 * 72 * 72
 UNIFORM GRID DIM(BIG): 72 * 72 * 72
 DONE(0.0612707  SEC) : SETUP UNITCELL
 DONE(0.157819   SEC) : SYMMETRY
 DONE(0.319416   SEC) : INIT K-POINTS
 ---------------------------------------------------------
 Cell relaxation calculations
 ---------------------------------------------------------
 SPIN    KPOINTS         PROCESSORS  
 1       196             16          
 ---------------------------------------------------------
 Use plane wave basis
 ---------------------------------------------------------
 ELEMENT NATOM       XC          
 Os      4           
 Pr      2           
 ---------------------------------------------------------
 Initial plane wave basis and FFT box
 ---------------------------------------------------------
 DONE(0.334265   SEC) : INIT PLANEWAVE
 MEMORY FOR PSI (MB)  : 93.173
 DONE(0.362569   SEC) : LOCAL POTENTIAL
 DONE(0.448029   SEC) : NON-LOCAL POTENTIAL
 DONE(0.584472   SEC) : INIT BASIS
 -------------------------------------------
 STEP OF RELAXATION : 1
 -------------------------------------------
 START CHARGE      : atomic

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 0 PID 950 RUNNING AT dp-lbg-389-6837361
=   KILLED BY SIGNAL: 11 (Segmentation fault)
===================================================================================
```

And the symmetry error is like:
```
 UNIFORM GRID DIM     : 72 * 72 * 72
 UNIFORM GRID DIM(BIG): 72 * 72 * 72
 DONE(0.108673   SEC) : SETUP UNITCELL
 DONE(0.201551   SEC) : SYMMETRY
match failed:
 
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                         NOTICE                           
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 symmetry operation of reciprocal lattice is wrong! 
 CHECK IN FILE : OUT.alloy/warning.log
 
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                         NOTICE                           
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

  |CLASS_NAME---------|NAME---------------|TIME(Sec)-----|CALLS----|AVG------|PER%-------
                       total               0.33898        5         0.068     1e+02     %
 ----------------------------------------------------------------------------------------
```
[Segmentation_fault_error.zip](https://github.com/deepmodeling/abacus-develop/files/11503317/Segmentation_fault_error.zip)
[Segmentation_fault_error_1.zip](https://github.com/deepmodeling/abacus-develop/files/11503319/Segmentation_fault_error_1.zip)
[Segmentation_fault_error_2.zip](https://github.com/deepmodeling/abacus-develop/files/11503325/Segmentation_fault_error_2.zip)
[Segmentation_fault_error_3.zip](https://github.com/deepmodeling/abacus-develop/files/11503329/Segmentation_fault_error_3.zip)
[Symmetry_error.zip](https://github.com/deepmodeling/abacus-develop/files/11503331/Symmetry_error.zip)

**Comment by Qianruipku** [comment]
The occurrence of "segmentation faults" may be due to insufficient memory. After rerunning these cases in other nodes, no faults were observed.

**Comment by maki49** [comment]
The symmetry part of promlem have been fixed in #2461 .
